### PR TITLE
04 review list

### PR DESCRIPTION
**Pull request by stormytuna**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,83 @@
...
+}
+
+.NavBar {
+  font-size: 20px;
+  font-weight: 400;
+  padding: 10px;
+  background-color: rgb(240, 240, 240);
+}
+
+.Review {
```
Comment (dav3rid) [inline src/App.css:28]: This content gets quite cramped and overlapped when viewing on a mobile size screen - consider implementing a structure (similar to what you have done for "ReviewInfo") like grid or flex.
Would also recommend viewing in a smaller view regularly when experimenting with CSS.

Comment (dav3rid) [reply src/App.css:28]: Other than this issue, a nice clean look!

File: src/components/NavBar.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { Link } from "react-router-dom";
+
+export function NavBar() {
+  return (
+    <div className="NavBar">
+      <Link to="/Reviews">Reviews</Link>;
```
Comment (dav3rid) [inline src/components/NavBar.jsx]: Can remove the rogue semi colon on this line - it is appearing on the webpage

File: src/components/Review.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      <div className="ReviewInfo">
+        <span className="Designer">by {designer}</span>
+        <span className="Category">{category}</span>
+        <span className="CreatedAt">{created_at}</span>
+        <span className="Owner">review by {owner}</span>
+      </div>
+
+      <span className="Votes">Votes: {votes}</span>
+      <span className="ReviewBody">{review_body}</span>
+      <img src={review_img_url} alt="" />
```
Comment (dav3rid) [inline src/components/Review.jsx]: Provide an `alt` value here!

File: src/components/Reviews.jsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { Review } from "./Review";
+
+export function Reviews() {
+  const [filters, setFilters] = useState({});
```
Comment (dav3rid) [inline src/components/Reviews.jsx]: Don't introduce state until it's necessary

File: src/components/Reviews.jsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  const [reviews, setReviews] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get("https://be-nc-games.onrender.com/api/reviews")
+      .then((response) => {
+        setReviews(response.data.reviews);
+        console.log(response.data.reviews[0]);
```
Comment (dav3rid) [inline src/components/Reviews.jsx]: remove your log here

File: src/components/Reviews.jsx [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+      })
+      .catch((e) => console.error(e));
```
Comment (dav3rid) [inline src/components/Reviews.jsx]: When catching errors, something visible to the user would be useful

File: src/components/Reviews.jsx [hunk 7]
```diff
@@ -0,0 +1,32 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { Review } from "./Review";
+
+export function Reviews() {
+  const [filters, setFilters] = useState({});
+  const [reviews, setReviews] = useState([]);
+
```
Comment (dav3rid) [inline src/components/Reviews.jsx:8]: Please make use of the `isLoading` pattern here to indicate to users on a slower connection that something is happening!

File: src/components/Reviews.jsx [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+      })
+      .catch((e) => console.error(e));
+  }, [filters]);
+
+  return (
+    <div className="Reviews">
+      <ul>
+        {reviews.map((review) => {
+          return (
+            <li>
```
Comment (dav3rid) [inline src/components/Reviews.jsx]: You're getting an error in the console - please add the key prop!

**Review by dav3rid — CHANGES_REQUESTED** [review]

**Comment by MRombi** [comment]
Well done on this ticket, please action on the feedback I mentioned and feel free to merge without resubmission 😄

**Pull request merged**